core/arch/arm/kernel/proc_a32.S

```diff
@@ -39,6 +40,7 @@
 
 /* void cpu_spin_lock(lock address) - lock mutex */
 FUNC cpu_spin_lock , :
+UNWIND(	.fnstart)
```

Comment:
No need to have them on?

    plat-sunxi/smp_fixup.S
    plat-sunxi/smp_boot.S
    plat-sunxi/entry.S
    plat-mediatek/mt8173_core_pos_a32.S
    kernel/misc_a32.S
    kernel/vfp_a32.S
    kernel/thread_a32.S
    kernel/generic_entry_a32.S
    kernel/ssvce_a32.S
    kernel/tz_ssvce_pl310_a32.S
    tee/arch_svc_a32.S
    sm/sm_a32.S
    plat-stm/asc.S
    plat-stm/tz_a9init.S
    plat-ls/ls_core_pos.S
    plat-vexpress/juno_core_pos_a32.S